src/test/resources/test_en.properties

```diff
@@ -0,0 +1,2 @@
+nls.test.withThree=zero|one|many: ${count}
```

Comment:
i dont think its very good to save 3 different translations under a single key.
since the versions are always the same wouldn't it be better to store them like this:

```
nls.test.withThree.zero=zero
nls.test.withThree.one=one
nls.test.withThree.many=many: ${count}
```

and make NLS fetch them using these suffixes?
Storing 3 different Translations does seem like bad practice to me and i can see problems when exporting such translations or confusion for the people translating these strings.

Reply:
this pattern is shamelessly stolen from other frameworks and it is almost a best practice to use one string...